<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>findStaff</class>
 <widget class="QMainWindow" name="findStaff">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>348</width>
    <height>441</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>查询员工</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton_back">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>360</y>
      <width>71</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>返回</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_clear">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>360</y>
      <width>71</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>清空</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_find">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>360</y>
      <width>71</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>查询</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_cardID">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>40</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>331</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);</string>
    </property>
    <property name="text">
     <string>请选择输入下面的任意几项:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_name">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>80</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_ID">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>120</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>40</y>
      <width>64</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>    卡号:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>80</y>
      <width>64</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>    姓名:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>116</y>
      <width>64</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string> 身份证号:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_sex">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>200</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string/>
     </property>
    </item>
    <item>
     <property name="text">
      <string>男</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>女</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="lineEdit_phone">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>160</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_department">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>240</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string/>
     </property>
    </item>
    <item>
     <property name="text">
      <string>董事会</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>销售部</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>技术部</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>后勤部</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="comboBox_position">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>280</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string/>
     </property>
    </item>
    <item>
     <property name="text">
      <string>董事长</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>总经理</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>经理</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>副经理</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>员工</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>156</y>
      <width>64</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>    电话:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>200</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>    性别:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>246</y>
      <width>64</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>    部门:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>290</y>
      <width>64</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>    职位:</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>348</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
